src/WebApiDebugger/Controllers/WeatherForecastController.cs

```diff
@@ -4,6 +4,7 @@ namespace WebApiDebugger.Controllers;
 
 [ApiController]
 [Route("[controller]")]
+[Produces("application/json")]
```

Comment:
Was this for testing purpose?

Reply:
Yup. Just making sure debug project doesn't generate any diff from Oasdiff